scripts/bespoke_input.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/local/bin/python
+
+import json
+import sys
+
+SRC = 1
+TGT = 2
+IP_PREF = 0
+TIME = 3
+
+class Update:
+    """
+    Internal representation of an individual BGP Update or Delta-net Update
```

Comment:
While the CSV format is syntactically is the same as Delta-net's, the semantics is very different, and I would therefore recommend removing references to Delta-net. Specifically, in your format the source and target denotes interfaces, whereas in the case of Delta-net they denote nodes.